### ISSUE TITLE
I am unable to connect my google drive. please check this error and correct it

### ISSUE THREAD

**Issue by surya-aezowie** [swiftyapp/swifty#19]
```
Error: invalid_grant
    at t.Gaxios.request (C:\Users\goldh\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:8:8812)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async u.getTokenAsync (C:\Users\goldh\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:3:20676)
```

---

Swifty 0.4.1
Electron 7.0.0
win32 10.0.18363
Locale: en-GB

**Comment by fusionarts** [comment]
Also the Chrome extension is not working. Maybe because it is not uploaded on Chrome Store! 
I have a developer account of Chrome web store so if you allow then i can publish the extension on  Chrome web store

**Comment by alchaplinsky** [comment]
Hey @surya-aezowie, try to connect Gdrive, and after you get this error, just click 'OK' and then click connect again. This worked for me. However this is an known issue and I'm going to work on fixing this asap.

**Comment by callmehanu** [comment]
I am also have problems to connect my **google drive**.
![image](https://user-images.githubusercontent.com/13230502/73680393-25b0db00-468a-11ea-8ee5-8f8ed37ab788.png)

**Comment by alchaplinsky** [comment]
Should be fixed in recent release [v0.4.4](https://github.com/swiftyapp/swifty/releases/tag/v0.4.4)
